### PR TITLE
#334 Added Player Colours to sources.json

### PR DESCRIPTION
**Pull request by McHearty**
Contrib yo

### CODE REVIEW

File: sources.json [hunk 1]
```diff
@@ -69,6 +69,18 @@
...
+    "2": "#5DE6AD",
+    "3": "#AD00AD",
+    "4": "#DCD90A",
+    "5": "#E66200",
+    "128": "#E67AB0",
+    "129": "#92A440",
+    "130": "#5CC5E0",
+    "131": "#00771F",
+    "132": "#956000"
+  }
```
Comment (albertcui) [inline sources.json]: Need a comma here:

```
},
```

**Comment by howardc93** [comment]
Should we call it player_colors or player_slot_colors, or does it not really matter? :P

**Comment by McHearty** [comment]
Player colours currently are only used in this way, so unless you want to future proof against any possible changes, player_colors should be sufficient

**Comment by howardc93** [comment]
sounds good.  signing off

**Pull request merged**